src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -608,6 +608,15 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
       onlyOneBlock
     );
     await this.#blockBeingSavedPromise;
+
+    if (
+      timestamp !== undefined &&
+      this.#options.miner.timestampIncrement === "clock"
```

Comment:
There are two time-related blocks of code in this function that both check `this.#options.miner.timestampIncrement === "clock"`. Can these two blocks be joined together? If not, can you cache `this.#options.miner.timestampIncrement === "clock"` in a variable so we don't do it twice?

Reply:
Good call!